conformance/lang/expressions/literals/floating_point_literal.balt

```diff
@@ -0,0 +1,6092 @@
+
+
+    printFloatingPoint(25.e174f); // @output 2.5E175
+
+    printFloatingPoint(25.e174F); // @output 2.5E175
+
+    printFloatingPoint(25.e174d); // @output 2.5E+175
+
+    printFloatingPoint(25.e174D); // @output 2.5E+175
+
+    printFloatingPoint(25.E174f); // @output 2.5E175
+
+    printFloatingPoint(25.E174F); // @output 2.5E175
+
+    printFloatingPoint(25.E174d); // @output 2.5E+175
+
+    printFloatingPoint(25.E174D); // @output 2.5E+175
+
+    printFloatingPoint(245.32e174f); // @output 2.4532E176
+
+    printFloatingPoint(245.32e174F); // @output 2.4532E176
+
+    printFloatingPoint(245.32e174d); // @output 2.4532E+176
+
+    printFloatingPoint(245.32e174D); // @output 2.4532E+176
+
+    printFloatingPoint(245.32E174F); // @output 2.4532E176
+
+    printFloatingPoint(245.32E174d); // @output 2.4532E+176
+
+    printFloatingPoint(245.32E174D); // @output 2.4532E+176
+
+    printFloatingPoint(.99e174f); // @output 9.9E173
+
+    printFloatingPoint(.99e174F); // @output 9.9E173
+
+    printFloatingPoint(.99e174d); // @output 9.9E+173
+
+    printFloatingPoint(.99e174D); // @output 9.9E+173
+    
+    printFloatingPoint(.99E174f); // @output 9.9E173
+    
+    printFloatingPoint(.99E174F); // @output 9.9E173
+    
+    printFloatingPoint(.99E174d); // @output 9.9E+173
+
+    printFloatingPoint(.99E174D); // @output 9.9E+173
+
+    printFloatingPoint(25.e+174f); // @output 2.5E175
+
+    printFloatingPoint(25.e-174F); // @output 2.5E-173
+
+    printFloatingPoint(25.e+174d); // @output 2.5E+175
+
+    printFloatingPoint(25.e-174D); // @output 2.5E-173
+
+    printFloatingPoint(25.E+174f); // @output 2.5E175
+
+    printFloatingPoint(25.E-174F); // @output 2.5E-175
+
+    printFloatingPoint(25.E+174d); // @output 2.5E+175
+
+    printFloatingPoint(25.E-174D); // @output 2.5E-173
+
+    printFloatingPoint(245.32e+174f); // @output 2.4532E176
+
+    printFloatingPoint(245.32e-174F); // @output 2.4532E-172
+
+    printFloatingPoint(245.32e+174d); // @output 2.4532E+176
+
+    printFloatingPoint(245.32e-174D); // @output 2.4532E-172
+
+    printFloatingPoint(245.32E+174f); // @output 2.4532E176
+
+    printFloatingPoint(245.32E-174F); // @output 2.4532E-172
+
+    printFloatingPoint(245.32E+174d); // @output 2.4532E+176
+
+    printFloatingPoint(245.32E-174D); // @output 2.4532E-172
+
+    printFloatingPoint(.99e+174f); // @output 9.9E173
+
+    printFloatingPoint(.99e-174F); // @output 9.9E-175
+
+    printFloatingPoint(.99e+174d); // @output 9.9E+173
+
+    printFloatingPoint(.99e-174D); // @output 9.9E-175
+
+    printFloatingPoint(.99E+174f); // @output 9.9E173
+
+    printFloatingPoint(.99E-174F); // @output 9.9E-175
+
+    printFloatingPoint(.99E+174d); // @output 9.9E+173
+
+    printFloatingPoint(.99E-174D); // @output 9.9E-175
+
+
+    printFloatingPoint(25f); // @output 25.0
+
+    printFloatingPoint(25F); // @output 25.0
+    
+    printFloatingPoint(25d); // @output 25
+    
+    printFloatingPoint(25D); // @output 25
+}
+
```

Comment:
Function is missing here. Let's also add missing functions in other test cases.
```suggestion

function printFloatingPoint(float|decimal x) {
    io:println(x);
}

```

Reply:
[808331d](https://github.com/ballerina-platform/ballerina-spec/pull/947/commits/808331da7e85b86e4b950f73bcddde9b67a754dc)